Package.swift

```diff
@@ -38,6 +38,12 @@ let package = Package(
                 "Yaml"
             ]
         ),
-        .testTarget(name: "LicensePlistTests", dependencies: ["LicensePlistCore"])
+        .testTarget(
+            name: "LicensePlistTests",
+            dependencies: ["LicensePlistCore"],
+            exclude: [
+                "XcodeProjects"
```

Comment:
I could exclude the test xcode-projects directory by specifying it in Package.swift.